.azure/gpu-tests-fabric.yml

```diff
@@ -74,7 +77,8 @@ jobs:
           echo "##vso[task.setvariable variable=COVERAGE_SOURCE]$scope"
         displayName: "set env. vars"
       - bash: |
-          echo "##vso[task.setvariable variable=TORCH_URL]https://download.pytorch.org/whl/test/cu${CUDA_VERSION_MM}/torch_test.html"
+          echo "##vso[task.setvariable variable=TORCH_URL]https://download.pytorch.org/whl/test/cu${CUDA_VERSION_MM}"
+          echo "##vso[task.setvariable variable=TORCHVISION_URL]https://download.pytorch.org/whl/test/cu124/torchvision-0.19.0%2Bcu124-cp311-cp311-linux_x86_64.whl"
```

Comment:
Why hard coded python version?

Reply:
It was the easiest way but if you want I can save the version into a env variable